DirectEmployers/MyJobs
# [PD 2815] adding legends to map

Task was to add legends to the maps.

To Test:
Pull down the repository. You can view the maps and see that the legends have been added at the top of the maps. There is also data associated with the tooltips now inside of the tooltip display if you hover over the maps. The cities view has been changed to a bar chart since we currently don't have a cities type of map to display so I figured a bar chart would be better suited.
